pkg/controller/directvolumemigration/rsync.go

```diff
@@ -1206,14 +1206,32 @@ func (t *Task) haveRsyncClientPodsCompletedOrFailed() (bool, bool, error) {
 					LastObservedProgressPercent: dvmp.Status.LastObservedProgressPercent,
 					LastObservedTransferRate:    dvmp.Status.LastObservedTransferRate,
 				})
+			case dvmp.Status.PodPhase == corev1.PodPending:
+				t.Owner.Status.PendingPods = append(t.Owner.Status.PendingPods, &migapi.PodProgress{
+					ObjectReference:             objRef,
+					LastObservedProgressPercent: dvmp.Status.LastObservedProgressPercent,
+					LastObservedTransferRate:    dvmp.Status.LastObservedTransferRate,
+				})
+				pendingPods = append(pendingPods, strings.Split(dvmp.Status.LogMessage, " ")[1])
```

Comment:
Failing to understand why this is not as simple as `obj.Name` and `object.Namespace`? What am I missing?

Reply:
Ahh, missed that obj has these two pieces of info.